tests/aiohttp/test_aiohttp_simple_api.py

```diff
@@ -214,3 +214,10 @@ def test_create_user(test_client, aiohttp_app):
     user = {'name': 'Maksim'}
     resp = yield from app_client.post('/v1.0/users', json=user, headers={'Content-type': 'application/json'})
     assert resp.status == 201
+
+
+@asyncio.coroutine
+def test_access_request_context(test_client, aiohttp_app):
```

Comment:
why is the parameter called `aiohttp_app` here? shouldn't it be the request context?